bundles/org.smarthomej.transform.math/src/main/java/org/smarthomej/transform/math/internal/AbstractMathTransformationService.java

```diff
@@ -34,27 +34,29 @@ abstract class AbstractMathTransformationService implements TransformationServic
 
     @Override
     public @Nullable String transform(String valueString, String sourceString) throws TransformationException {
-        QuantityType<?> source;
-        try {
-            source = new QuantityType<>(sourceString);
-        } catch (IllegalArgumentException e) {
-            logger.warn("Input value '{}' could not be converted to a valid number", sourceString);
-            throw new TransformationException("Math Transformation can only be used with numeric inputs");
-        }
-        QuantityType<?> value;
-        try {
-            value = new QuantityType<>(valueString);
-        } catch (IllegalArgumentException e) {
-            logger.warn("Input value '{}' could not be converted to a valid number", valueString);
-            throw new TransformationException("Math Transformation can only be used with numeric inputs");
+        
+        if("NULL".equals(valueString) || "NULL".equals(sourceString)) {
```

Comment:
Wouldn't it make sense to check `UNDEF`, too?

Reply:
@clinique Can you comment here? I would like to do a release in the next days due to #490.

Reply:
> Wouldn't it make sense to check `UNDEF`, too?

You're correct, I thought of it but I only faced the error with NULL, never UNDEF, so thought : if not broken, don't fix it :)